opentelemetry-instrumentation/pyproject.toml

```diff
@@ -26,8 +26,9 @@ classifiers = [
 ]
 dependencies = [
   "opentelemetry-api ~= 1.4",
-  "setuptools >= 16.0",
   "wrapt >= 1.0.0, < 2.0.0",
+  "importlib-metadata >= 6.0, < 8.0",
```

Comment:
We already vendor importlib-metadata in opentelemetry-api (https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-api/src/opentelemetry/util/_importlib_metadata.py) - should we reuse this rather than re-introduce the dependency?

Reply:
https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2181#discussion_r1739214916